banzai/qc/header_checker.py

```diff
@@ -45,144 +45,99 @@ def do_stage(self, images):
 
        """
         for image in images:
-
-            self.check_header_keyword_present(self.header_expected_format.keys(), image)
-            # Remove for now, until we can re-evaluate what exactly the
-            # expected formats should be. There could be multiple expected
-            # formats per keyword.
-            # self.check_header_format(keyword, image)
-            self.check_header_na(self.header_expected_format.keys(), image)
-            self.check_ra_range(image)
-            self.check_dec_range(image)
-            self.check_exptime_value(image)
-
+            bad_keywords = self.check_keywords_missing_or_na(image)
+            self.check_ra_range(image, bad_keywords)
+            self.check_dec_range(image, bad_keywords)
+            self.check_exptime_value(image, bad_keywords)
         return images
 
-    def check_header_keyword_present(self, keywords, image):
-        """ Logs an error if the keyword is not in the image header.
+    def check_keywords_missing_or_na(self, image):
+        """ Logs an error if the keyword right_ascension is not inside
+            the expected range (0<ra<360 degrees) in the image header.
```

Comment:
Is this supposed to be here in the comment? ra seems like it would be somewhere else.

Reply:
Fixed